backend/api/serializers.py

```diff
@@ -74,13 +88,51 @@ class Meta:
             "auditee_fiscal_period_end",
         ]
 
+    # auditee_name and auditee_uei optional, fiscal start/end required
+    def validate_auditee_fiscal_period_start(self, value):
+        if not value:
+            raise serializers.ValidationError(AUDITEE_FISCAL_PERIOD_START)
+        return value
+
+    def validate_auditee_fiscal_period_end(self, value):
+        if not value:
+            raise serializers.ValidationError(AUDITEE_FISCAL_PERIOD_END)
+        return value
+
 
 class AccessSerializer(serializers.ModelSerializer):
     class Meta:
         model = Access
         fields = ["role", "email", "user_id"]
 
 
+class AccessAndSubmissionSerializer(serializers.Serializer):
+    """
+    The data here should be a series of role/email pairs.
+    We want to verify that there is one and only one certifying_auditee_contact
+    and one and only one certifying_auditor_contact listed.
```

Comment:
I think the language here should be updated based on the change to the data structure (no longer a series of role/email pairs, new structure doesn't require us to validate a single certifier, etc)

Reply:
I can probably just remove this doc block as the other Serializers don't have them, but also the fields listed below it pretty clearly describe what the data coming in should look like.